src/high-level/src/KinDynComputations.cpp

```diff
@@ -1798,6 +1798,65 @@ SpatialMomentum KinDynComputations::getCentroidalTotalMomentum()
     return newOutputFrame_X_oldOutputFrame*base_momentum;
 }
 
+bool KinDynComputations::getCentroidalTotalMomentumJacobian(MatrixDynSize& centroidalMomentumJacobian)
+{
+    if (!getLinearAngularMomentumJacobian(centroidalMomentumJacobian))
+        return false;
+
+    // get CoM and base pose
+    Position com = getCenterOfMassPosition();
+    const Position& basePosition = pimpl->m_pos.worldBasePos().getPosition();
+    const Rotation& A_R_B = pimpl->m_pos.worldBasePos().getRotation();
+
+    if (pimpl->m_frameVelRepr == BODY_FIXED_REPRESENTATION)
```

Comment:
Would a switch-case statement be better in this case? If you cover all the cases of an enum omitting the default case, modern compilers issue a warning if a new case is added, allowing to spot all the points in the code where the new case is not considered.

Reply:
Done in 3feae00a6cd2fc3d7a71018f86a56fca44cf1c62